test/skimNtuple2018.cpp

```diff
@@ -3791,7 +3791,7 @@ int main (int argc, char** argv)
                   else if (tlv_secondLepton.Pt() < 200) SFTau2 = 0.91;
                   else                                  SFTau2 = 0.97;
 
-                  float VBFtrigNorm = 1.;   // No need to normalize for 2018
+                  float VBFtrigNorm = 0.990342;   // from: https://twiki.cern.ch/twiki/bin/viewauth/CMS/DoubleHiggsToBBTauTauWorkingLegacyRun2#Triggers
```

Comment:
mmm boh

Reply:
yeah i'm not 100% sure either, I took it from the twiki. If you prefer I can put it back to 1. It shouldn't change much anyway